### PR TITLE
Chore/update stress tests

### PR DESCRIPTION
**Pull request by mesudip**
## List of changes

- Update test scenario to use current apis used in govtool frontend.
- Replace random ids with actual drep and proposal ids for seeding.

## Checklist

- [related issue](https://github.com/IntersectMBO/govtool/issues/1107)
- [x] My changes generate no new warnings
- [x] My code follows the [style guidelines](https://github.com/IntersectMBO/govtool/tree/main/docs/style-guides) of this project
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the [changelog](https://github.com/IntersectMBO/govtool/blob/main/CHANGELOG.md)
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: tests/load-testing/src/test/java/org/cardano/govtool/simulations/VvaSimulation.java [hunk 1]
```diff
@@ -36,43 +52,23 @@ public void before() {
...
-                )
+                  makeScenario("User Connects and Leave", exec(), 0.1)
+                , makeScenario("User Registers as Drep",
+                        exec(DRepAction.registerAsDRep), 0.1),
+                 makeScenario("User Views Proposals",
+                        exec(Action.viewProposals), 0.2)
+                , makeScenario("AdaHolder delegates to Drep",
+                        exec(AdaHolderAction.delegateToDRep), 0.1)
+                , makeScenario("Drep votes on Proposal",
+                        exec(DRepAction.vote), 0.3 * 0.4)
```
Comment (NabinKawan) [inline tests/load-testing/src/test/java/org/cardano/govtool/simulations/VvaSimulation.java:66]: Please, Extract ` (0.3 * 0.4)` into variable.

File: tests/load-testing/src/test/java/org/cardano/govtool/simulations/VvaSimulation.java [hunk 2]
```diff
@@ -36,43 +52,23 @@ public void before() {
...
+                , makeScenario("Drep view Votes",
+                        exec(Action.viewProposals).pause(2).exec(Action.viewProposals), 0.3 * 0.5)
```
Comment (NabinKawan) [inline tests/load-testing/src/test/java/org/cardano/govtool/simulations/VvaSimulation.java:68]: Please, Extract  (0.3 * 0.5) into variable.

File: tests/load-testing/src/test/java/org/cardano/govtool/simulations/VvaSimulation.java [hunk 3]
```diff
@@ -36,43 +52,23 @@ public void before() {
...
+                , makeScenario("Drep Retirement",
+                        exec(DRepAction.retireAsDRep), 0.3 * 0.1)
```
Comment (NabinKawan) [inline tests/load-testing/src/test/java/org/cardano/govtool/simulations/VvaSimulation.java:70]: Please, Extract  (0.3 * 0.1) into variable.

**Review by NabinKawan — CHANGES_REQUESTED** [review]
Looks great! I've requested some changes.

File: tests/load-testing/src/test/java/org/cardano/govtool/simulations/VvaSimulation.java (hunk 1 above)
Comment (mesudip) [reply tests/load-testing/src/test/java/org/cardano/govtool/simulations/VvaSimulation.java]: Done

File: tests/load-testing/src/test/java/org/cardano/govtool/simulations/VvaSimulation.java (hunk 3 above)
Comment (mesudip) [reply tests/load-testing/src/test/java/org/cardano/govtool/simulations/VvaSimulation.java]: Done

File: tests/load-testing/src/test/java/org/cardano/govtool/simulations/VvaSimulation.java (hunk 2 above)
Comment (mesudip) [reply tests/load-testing/src/test/java/org/cardano/govtool/simulations/VvaSimulation.java]: Done

**Review by NabinKawan — APPROVED** [review]
Thanks, LGTM

**Pull request merged**